internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestVeniceHelixAdminWithIsolatedEnvironment.java

```diff
@@ -42,7 +42,7 @@
 public class TestVeniceHelixAdminWithIsolatedEnvironment extends AbstractTestVeniceHelixAdmin {
   @BeforeMethod(alwaysRun = true)
   public void setUp() throws Exception {
-    setupCluster(false);
+    setupCluster(false, new MetricsRepository());
```

Comment:
same as above.

Reply:
It is removed in `AbstractTestVeniceHelixAdmin`. Again, I don't have any concerns in reverting it. Let me know whatever works.

Reply:
Feel free to resolve the similar comment after responding to original thread. Marking this as resolved.